client/src/hooks/useDownloadOptionsContext.js

```diff
@@ -114,9 +127,12 @@ export const useDownloadOptionsContext = () => {
     computedFile,
     computedFiles,
     getFoundFile,
+    isMergedObjectsAvailable,
     getOptionsAndDefault,
     saveUserPreferences,
     resourceSort,
-    resource
+    resource,
+    selectedMerged,
+    setSelectedMerged
```

Comment:
```suggestion
    isMerged,
    setIsMerged
```
Lets keep the convention of using the attribute name as the state variable name here.

Reply:
This state is used to track whether the merged objects checkbox is selected or not, and we typically use the prefix `selected`. 

Also I used this name as suggested in the linked issue #550. Please let me know!

Reply:
I renamed it to `includesMerged` as you suggested at 9763825ca84011c45856df985020f19b48e5f1a9 via Eng meeting, thank you!